packages/client/README.md

```diff
@@ -40,6 +40,45 @@ if (updatedFrameAfterPost.buttons['1'].action === 'post_redirect') {
 }
 ```
 
+### Transaction frame usage
+
+```ts
+import { readMetadata, post, postTransaction } from '@open-frames/proxy-client';
+
+const FRAME_URL = 'https://frames.xmtp.chat/';
+
+const initialFrame = await readMetadata(FRAME_URL);
+
+// Extract from above
+const action = initialFrame.buttons['1'].action;
+const target = initialFrame.buttons['1'].target;
+const postUrl = initialFrame.buttons['1'].post_url;
+
+// If this is a transaction frame, do the below steps
+const isTransaction = action === 'tx' && target && postUrl;
+
+// First, make a POST request to the `target` URL to fetch data about the transaction, with a signed frame action payload in the POST body including the address of the connected wallet in the `address` field.
+const fetchTransactionData: {
+	chainId: string;
+	method: 'eth_sendTransaction';
+	params: {
+		abi: Abi | [];
+		to: `0x${string}`;
+		value?: string;
+		data?: `0x${string}`;
+	};
+} = await postTransaction(target, { address: `0x${string}` });
```

Comment:
I'm not sure if `postTransaction` is really the appropriate var name here, given the fact that it's really the call to get the transaction data. However, the logic inside this call essentially the same as the `post` call just with different types so that the user doesn't have to do any type assertions, so it felt strange to be essentially the same method but with a very different naming convention. Would love any feedback here.

Reply:
Hmmm. I see your point. It's a little confusing because there are two posts related to transactions. One to get the info and then the regular post that includes the `transactionId` (which can be accomplished using the regular POST method).

Makes me wonder if we should name this something like `transactionInfo` just to disambiguate the two.

Weakly held opinion.

Reply:
Thanks! Also ran this by a few folks and seems no one finds this particularly confusing either. So I'll just leave it as-is and we can always update if needed in the future.